Documentation/readme.md

```diff
@@ -1,4 +1,57 @@
 # Documentation
+This is an Web-Api project that is based on the Starwars Web API, https://swapi.dev/ and is used to for practicing Http-Requests, EF and ASP.NET web framework.
 
-Use this file to fill in your documentation
+## [Project SpacePark-API](https://github.com/PGBSNH20/spaceparkv2-buddygroup6-renegades/tree/main/Source/SpacePark-API):
+This is the main project and contains code regarding:
+- Data Models
+- DbContext
+- Controllers
+- Logic
 
+How we work ***Server side*** with token and identification, authorization, we have two roles - admin and user.
+```csharp
+if (account.Password != model.Password || account.AccountName != model.Username) return Unauthorized();
+            var identity = GetClaimsIdentity(account);
+            var token = GetJwtToken(identity);
+            var tokenString = new JwtSecurityTokenHandler().WriteToken(token);
+            var userToken = new UserToken()
+            {
+                Account = account,
+                ExpiryDate = token.ValidTo,
+                Token = tokenString
+            };
+            _repository.Add(userToken);
+            _repository.SaveChanges();
+            return Ok(new
+            {
+                token = tokenString,
+                expiration = token.ValidTo
+            });
+```
+And ***Client side***
+```csharp
+public async Task<bool> LoginAsync(string accountName, string password)
+{
+var client = new HttpClient();
+var response = await JsonPostRequestAsync(EndPoints.Account.login, new
+{
+username = accountName,
+password = password
+});
+var responseContentString = await response.Content.ReadAsStringAsync();
+var token = GetResponseValueWithKey("token", responseContentString);
+_client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+
+ return response.IsSuccessStatusCode;
+}
+```
+
+## [Poject: SpaceParkTests](https://github.com/PGBSNH20/spaceparkv2-buddygroup6-renegades/tree/main/Source/SpaceParkTests):
+This is an x-unit project that do all the testing of the application.
+We use an in-memory-sql database (dependent on nu-get EntityFrameworkCore.InMemory) to run tests of the db-context and we also use an test-unique web-host for testing the http-requests. See [MockWebHostFactory.cs](https://github.com/PGBSNH20/spaceparkv2-buddygroup6-renegades/blob/main/Source/SpaceParkTests/MockWebHostFactory.cs)
+
```

Comment:
I think MockWebHostFactory.cs is gone because I got a "page not found"

Really nice you guys linked to each document so you can click on them and it redirects you! 🥇